openapi.yaml

```
@@ -58,26 +54,30 @@ paths:
               schema:
                 $ref: "#/components/schemas/Problem"
 
-  /validate:
+  /parse:
     post:
-      summary: Validate the given AsyncAPI document.
+      summary: Parse the given AsyncAPI document.
       operationId: validate
```

how about a different id 🙂?
```suggestion
      operationId: parse
```